src/rust/ide/view/graph-editor/src/component/visualization/container.rs

```diff
@@ -185,8 +184,8 @@ ensogl::define_endpoints! {
 pub struct View {
     logger         : Logger,
     display_object : display::object::Instance,
-    // background     : component::ShapeView<background::Shape>,
-    overlay        : component::ShapeView<overlay::Shape>,
+    // background     : background::View,
```

Comment:
This is commented code: there _is_ a TODO for that, but in line 58. I would add a reference to it, otherwise someone eagerly delete this line in future.